src/test/java/org/imsglobal/caliper/partner/events/RenLearnConstants.java

```diff
@@ -0,0 +1,9 @@
+package org.imsglobal.caliper.partner.events;
+
+import org.imsglobal.caliper.ri.events.HMHConstants;
+
+public class RenLearnConstants extends HMHConstants{
```

Comment:
good move extending HMHConstants - we should try to keep that HMHConstants for common fields only 👍

Reply:
Fixed

Reply:
Thanks. We can move the HMHConstants out of `ri` package also and create new constants extend from HMHConstants for ri tests